docs/assemblies/assembly-prometheus-metrics-proxy.adoc

```diff
@@ -0,0 +1,92 @@
+// file included in the following:
+//
+// kroxylicious-operator/index.adoc
+
+[id='assembly-prometheus-metrics-proxy-{context}']
+= Overview of Proxy Metrics
+
+[role="_abstract"]
+
```

Comment:
Need to think about where record encryption metrics are documented.  I think those will belong in the record encryption book.  I think this section needs to mention that filters might emit their own metrics and you should refer to their documentation for details.